features_and_tools/firely-server-ingest.rst

```diff
@@ -112,144 +117,379 @@ The execution of FSI can be configured using input parameters. These parameters
+
+Source
+^^^^^^
+
+
+* ``--sourceType <Filesystem|MongoDb|None>``: 
+
+  * **Config**: sourceType
+  * **Required**: No
+  * **Description**: Specifies the source type. None will try only to provision the target database.
+  * **Options**:
+
+    * **Filesystem**: read data from the filesystem
+    * **MongoDb**: read data from a Firely Server MongoDB database 
+    * **None**: use this option if you only want to provision the target database
+
+Source (for Filesystem)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* ``-s``, ``--source <source>``: 
+
+  * **Config**: sourceType
+  * **Required**: Yes
+  * **Description**: Input directory for work (this directory is visited recursively including all the subdirectories).
+
+Source (for MongoDb)
+^^^^^^^^^^^^^^^^^^^^
+
+This source is intended to be used in zero-downtime migration scenarios. Currently, it is only possible to use another MongoDB database as the target database.
+
+See more information on how to run migrations in :ref:`this article <zero_downtime_migration>`.
+
+* ``--srcMongoConnectionString <srcMongoConnectionString>``: 
+
+  * **Config**: mongoDbSource/connectionString
+  * **Required**: Yes
+  * **Description**: Connection string to read resources from.
+
+* ``--srcMongoCollection <srcMongoCollection>``: 
+
+  * **Config**: mongoDbSource/collectionName
+  * **Required**: Yes
+  * **Description**: Collection name to read entries from.
+
+* ``--srcMongoRunningMode <AdHoc|Continuous>``: 
+
+  * **Config**: mongoDbSource/runningMode
+  * **Required**: No
+  * **Description**: The mode in which the application should run.
+  * **Options**:
+
+    * **AdHoc** (default): the application will run once and exit
+    * **Continuous**: the application will run continuously and listen for changes in the source database until terminated by the user
+
+* Documents filter (can be set only via the config file): 
+
+  * **Config**: mongoDbSource/documentFilterBson
+  * **Required**: No
+  * **Description**: BSON filter to apply when reading documents. See `MongoDB aggregation match syntax <https://www.mongodb.com/docs/manual/reference/operator/aggregation/match/>`_ for details.
+
+
+Target
+^^^^^^
+
+* ``--dbType <MongoDb|SQL>``: 
+
+  * **Config**: databaseType
+  * **Required**: No
+  * **Description**: Specifies the target database type.
+  * **Options**:
+
+    * **SQL**
+    * **MongoDb**
+
+
+* ``--provisionTargetDatabase <true|false>``: 
+
+  * **Config**: provisionTargetDatabase
+  * **Required**: No
+  * **Description**: Whether to provision the target database. *Note: currently only supported for MongoDB.*
+
+Target (for SQL Server)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* ``-c``, ``--connectionstring <connectionstring>``: 
+
+  * **Config**: sqlServer/connectionString
+  * **Required**: Yes
+  * **Description**: Connection string to Firely Server SQL Server database.
+
+* ``--sqlPar <sqlPar>``: 
+
+  * **Config**: sqlServer/saveParallel
+  * **Required**: No
```

Comment:
It was not in the original text either, but it would be helpful to add the default value. Since it is not required, but some value will be used, let's inform the user what that value is.